.circleci/config.yml

```diff
@@ -264,6 +264,23 @@ jobs:
       - simulate_nested:
           task: "/sep/zora-002-fp-upgrade"
 
+  simulate_sequence:
+    description: "Runs a sequence of simulations"
+    parameters:
+      prefix:
+        type: string
+      tasks:
+        type: string
+      block_number:
+        type: string
+    docker:
+      - image: <<pipeline.parameters.default_docker_image>>
+    steps:
+      - utils/checkout-with-mise
+      - run:
+          name: "Run simulation sequence for <<parameters.prefix>> tasks: <<parameters.tasks>>"
+          command: export FOUNDRY_DISABLE_NIGHTLY_WARNING=0; git submodule update; just clean; just install && echo "yes" | bash ./script/utils/sim-sequence.sh <<parameters.prefix>> "<<parameters.tasks>>" "<<parameters.block_number>>"
```

Comment:
We now use mise and foundry v1 in this repo, so we should be able to remove `export FOUNDRY_DISABLE_NIGHTLY_WARNING=0`

Reply:
Got it! 
I did this because I had warnings logs that was ugly into CI simulation. 
will do that now and see.